source/plugin/Assets/GoogleMobileAds/Common/DummyClient.cs

```diff
@@ -14,19 +14,42 @@
 
 using System;
 using System.Reflection;
+using System.Collections.Generic;
+using System.Threading.Tasks;
 
 using GoogleMobileAds.Unity;
 using GoogleMobileAds.Api;
+
 using UnityEngine;
+using UnityEngine.UI;
 
 namespace GoogleMobileAds.Common
 {
     public class DummyClient : IBannerClient, IInterstitialClient, IRewardBasedVideoAdClient,
             IAdLoaderClient, IMobileAdsClient
     {
+        private bool isOpen;
```

Comment:
can we move all the relevant banner client code into a new banner client to improve code readability?

We can also create a `Unity.Utils` class for any common utility code.